README.md

```diff
@@ -322,6 +324,8 @@ The default port is 1337, to use a different port set the PORT environment varia
 $ PORT=8080 parse-server --appId APPLICATION_ID --masterKey MASTER_KEY
 ```
 
+Must set `PARSE_PUBLIC_SERVER_URL` equal to `PARSE_SERVER_URL` if using HTTPS ([details here](https://github.com/ParsePlatform/parse-server/issues/2743)).
```

Comment:
> kindly specify what it should or should not be set to

This is what I just did. It's documented here: http://parseplatform.github.io/docs/parse-server/guide/#configuring-file-adapters